opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -226,31 +262,37 @@ def __init__(
             description,
             unit,
             value_type,
+            meter,
             label_keys=label_keys,
             enabled=enabled,
             monotonic=monotonic,
         )
 
-    def record(
-        self, label_values: Sequence[str], value: metrics_api.ValueT
-    ) -> None:
+    def record(self, label_set: LabelSet, value: metrics_api.ValueT) -> None:
         """See `opentelemetry.metrics.Measure.record`."""
-        self.get_handle(label_values).record(value)
+        self.get_handle(label_set).record(value)
 
     UPDATE_FUNCTION = record
 
 
+# Singleton of meter.get_label_set() with zero arguments
```

Comment:
Nit: this is evident from the code, the comment would be more helpful if it described how this is used. I actually expected it to be a placeholder default arg like `Tracer.CURRENT_SPAN`.